botmother/management/commands/polling.py

```diff
@@ -0,0 +1,28 @@
+import json
+import time
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from botmother.utils.api import TelegramAPI
+from botmother.utils.bot_router import BotRouter
+from example.urls import dispatch
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        telegram = TelegramAPI(settings.BOT_TOKEN)
+        updates_id = []
+
+        while True:
+            updates = telegram.get_updates()
+
+            for update in updates['result']:
+                if len(updates_id) == 0:
+                    updates_id.append(update['update_id'])
+
+                if update['update_id'] not in updates_id:
+                    updates_id.append(update['update_id'])
+                    router = BotRouter(json.dumps(update))
+                    dispatch(router)
```

Comment:
Import dispatch from address string in settings

Reply:
import importlib

module_name, func_name = settings.BOT_POOLING_DISPATCHER.rsplit('.', 1)
module = importlib.import_module(module_name)
dispatch = getattr(module, func_name)

Solved like that and it's working others I'll see